.pipelines/restore-acquiring_code.cmd

```diff
@@ -0,0 +1,89 @@
+@echo off
+
+REM Prerequisities
+echo.
+echo Deleteing python from path...
+set PYTHONPATH=
+set PYTHONHOME=
+
+echo.
+echo Installing Visual Stuio 2019 workloads...
+c:
+mkdir c:\Downloads
+cd c:\Downloads
+curl -o vs_enterprise.exe -L "https://aka.ms/vs/16/release/vs_enterprise.exe"
+call C:\Downloads\vs_enterprise.exe modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise" --add Microsoft.VisualStudio.Component.VC.ATLMFC;includeRecommended --quiet --wait
+
+echo.
+echo Installing Windows Debugging Tools...
+curl -o winsdksetup.exe -L "https://go.microsoft.com/fwlink/p/?linkid=2083338&clcid=0x409"
+winsdksetup.exe /features OptionId.WindowsDesktopDebuggers /quiet
+
+REM Getting depot_tools
+echo.
+echo Cloneing the depot_tools repo in the root of the c:drive...
+c:
+cd \
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools
+
+echo.
+echo Switching to the m80 branch of depot_tools...
+cd depot_tools
+git checkout chrome/3987
+git fetch
+
+echo.
+echo Downloading gclient...
+echo N | gclient
+
+echo.
+echo Setting the path environment variable...
+set PATH=c:\depot_tools;%PATH%
+
+REM Installing Python 2.7
+echo.
+echo Making sure that any python program executed in this session comes from the depot_tools...
+where python
+
+echo.
+echo Installing pip...
+call python -m pip install --upgrade pip
```

Comment:
Can the script check if the right python is being called, log an error message and stop execution?

Reply:
The previous command "where python" confirmed the right python location.

Reply:
Thanks for adding that check.